src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java

```diff
@@ -49,12 +49,15 @@ public void testGetMicrosoftTimeZoneNameBad() {
     // null-argument is not allowed.
     try {
       Assert.fail(TimeZoneUtils.getMicrosoftTimeZoneName(null));
-    } catch (final IllegalArgumentException ignored) {}
-
-    // Case-insensitive ID is not supported.
-    checkGetMicrosoftTimeZoneName("africa/abidjan", "UTC");
+    } catch (final IllegalArgumentException ignored) {
+    }
   }
 
+  @Test
+  public void testCaseInsensitiveIdNotSupported() {
+    // Case-insensitive ID is not supported so the map look up returns null.
+    checkGetMicrosoftTimeZoneName("africa/abidjan", null);
```

Comment:
I am able to reproduce this test failing when I use Java 8. However, I'm still not able to explain why it returns UTC when ```"africa/abidjan", "UTC"``` clearly does not exist in the map.

-shrug-